lib/commands/status.js

```diff
@@ -42,22 +42,29 @@ module.exports = {
 
 		const promises = [
 			execCommand.execute( getExecData( 'git rev-parse HEAD' ) ),
-			execCommand.execute( getExecData( 'git status --branch --porcelain' ) )
+			execCommand.execute( getExecData( 'git status --branch --porcelain' ) ),
+			execCommand.execute( getExecData( 'git describe --abbrev=0 --tags' ) ),
+			execCommand.execute( getExecData( 'git tag --sort=committerdate' ) )
```

Comment:
```
"mrgit": "https://github.com/cksource/mrgit.git@latest"
```

<img width="619" alt="image" src="https://user-images.githubusercontent.com/2270764/205148233-0b74f17f-8b08-46c8-b5eb-9f32ba9e0fc1.png">

This is crazy. We need to find a new way to extract the latest tag.

Reply:
Addressed in 0b9fc08.